packages/components/src/components/route-with-subroutes/route-with-subroutes.tsx

```diff
@@ -74,7 +91,7 @@ const RouteWithSubRoutes = ({
             );
         }
 
-        const title = getTitle?.() || '';
+        const title = getTitle?.() ?? '';
```

Comment:
@likhith-deriv Will the title have the same resulting value with || here 😊?

Reply:
yes. The difference between || & ?? is that, `||` operator checks for falsy. where as `??` is the fallback operator if the expression evaluates undefined or null

Reply:
@likhith-deriv yeah, i understand 😄 but the thing is I just can't figure out why `??` is more suitable here than `||`. It seems like if we leave it to be `||`, we'll still have an empty string as a result here in case `getTitle?.()` returns any falsy value including either an empty string, or undefined, or null. But with `??`, in case _hypothetically_ `getTitle?.()` returns `false`, **title** will be `false`, or if `getTitle?.()` returns `0`, then **title** will be `0`, and if `getTitle?.()` returns an empty string, it won't make much difference because we'll return an empty string just like with `||`, right?😊

Reply:
I thought it's better to be specific with what is expected rather than rely on JS. I am  not sure what's the right way to do. If || is to be used as fallback, I'll make the change